File: Tl2.3 RESTFULL_JAVA_EUREKA_BANK_GR08/02.CLICON/CLICON_EUREKA_GR08/src/main/java/ec/edu/monster/model/MovimientoModel.java
/*
 * Click nbfs://nbhost/SystemFileSystem/Templates/Licenses/license-default.txt to change this license
 * Click nbfs://nbhost/SystemFileSystem/Templates/Classes/Class.java to edit this template
 */
package ec.edu.monster.model;

/**
 *
 * @author crist
 */
public class MovimientoModel {
    private String codigoCuenta;
    private int numeroMovimiento;
    private String fechaMovimiento;
    private String codigoEmpleado;
    private String codigoTipoMovimiento;
    private String tipoDescripcion;
    private double importeMovimiento;
    private String cuentaReferencia;
    private double saldo; // Saldo acumulativo después del movimiento

    // Getters and setters
    public String getCodigoCuenta() {
        return codigoCuenta;
    }

    public void setCodigoCuenta(String codigoCuenta) {
        this.codigoCuenta = codigoCuenta;
    }

    public int getNumeroMovimiento() {
        return numeroMovimiento;
    }

    public void setNumeroMovimiento(int numeroMovimiento) {
        this.numeroMovimiento = numeroMovimiento;
    }

    public String getFechaMovimiento() {
        return fechaMovimiento;
    }

    public void setFechaMovimiento(String fechaMovimiento) {
        this.fechaMovimiento = fechaMovimiento;
    }

    public String getCodigoEmpleado() {
        return codigoEmpleado;
    }

    public void setCodigoEmpleado(String codigoEmpleado) {
        this.codigoEmpleado = codigoEmpleado;
    }

    public String getCodigoTipoMovimiento() {
        return codigoTipoMovimiento;
    }

    public void setCodigoTipoMovimiento(String codigoTipoMovimiento) {
        this.codigoTipoMovimiento = codigoTipoMovimiento;
    }

    public String getTipoDescripcion() {
        return tipoDescripcion;
    }

    public void setTipoDescripcion(String tipoDescripcion) {
        this.tipoDescripcion = tipoDescripcion;
    }

    public double getImporteMovimiento() {
        return importeMovimiento;
    }

    public void setImporteMovimiento(double importeMovimiento) {
        this.importeMovimiento = importeMovimiento;
    }

    public String getCuentaReferencia() {
        return cuentaReferencia;
    }

    public void setCuentaReferencia(String cuentaReferencia) {
        this.cuentaReferencia = cuentaReferencia;
    }

    public double getSaldo() {
        return saldo;
    }

    public void setSaldo(double saldo) {
        this.saldo = saldo;
    }
}

File: Tl2.2 SOAP_DOTNET_EUREKA_BANK_GR08/01.SERVIDOR/ws_eureka_bank_soap_dotnet/ec.edu.monster.model/MovimientoModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Runtime.Serialization;
using System.Text;
using System.Threading.Tasks;

namespace ws_eureka_bank_soap_dotnet.ec.edu.monster.model
{
    [DataContract]
    public class MovimientoModel
    {
        [DataMember]
        public string CodigoCuenta { get; set; }
        [DataMember]
        public int NumeroMovimiento { get; set; }
        [DataMember]
        public DateTime? FechaMovimientoDt { get; set; }
        [DataMember]
        public string FechaMovimiento { get; set; } // String para coincidir con Java
        [DataMember]
        public string CodigoEmpleado { get; set; }
        [DataMember]
        public string CodigoTipoMovimiento { get; set; }
        [DataMember]
        public string TipoDescripcion { get; set; }
        [DataMember]
        public double ImporteMovimiento { get; set; }
        [DataMember]
        public string CuentaReferencia { get; set; }
        [DataMember]
        public double Saldo { get; set; }
    }
}
